modes.xml

```diff
@@ -143,11 +146,14 @@
         <file name="spa-cat_valencia_gva.t1x.bin"/>
       </program>
       <program name="apertium-posttransfer"/>
-      <program name="lt-proc $1">
+      <program name="lt-proc $1 -b">
         <file name="spa-cat_valencia_gva.autogen.bin"/>
       </program>
-      <program name="lt-proc -x">
-        <file name="spa-cat.autopgen-diacritics-vells.bin"/>
+      <program name="cg-proc" debug-suff="pre2017prefs">
+        <file name="spa-cat.pre2017_prefs.rlx.bin"/>
```

Comment:
I guess this is a pipeline with hardcoded prefs (like my nob-nno_e)?

Reply:
Yes, this is one forcing the old ortography (it's easier having a "mode" for that than always passing the variable everywhere)